docs/how-to-guides/new-user-guides/launch-kubernetes-with-rancher/use-new-nodes-in-an-infra-provider/vsphere/create-a-vm-template.md

```diff
@@ -23,7 +23,7 @@ If you have any specific firewall rules or configuration, you will need to add t
 
 ## Linux Dependencies
 
-The packages that need to be installed on the template are listed below. These will have slightly different names based on distribution; some distributions ship these by default, for example.
+The packages that need to be installed on the template are listed below. These will have slightly different names based on distribution; for example, some distributions ship these dependencies by default. The dependencies listed here are required for the functioning of the Rancher cluster provisioner. The cluster provisioner automatically installs additional dependencies required for Kubernetes:
```

Comment:
> The cluster provisioner automatically installs additional dependencies required for Kubernetes:

Ending that sentence with a colon makes it look like the listed dependencies are the ones required for Kubernetes. The whole point of this PR is to clarify that they are not. I'll fix this!